webrender/src/renderer.rs

```diff
@@ -171,6 +171,49 @@ const GPU_SAMPLER_TAG_TRANSPARENT: GpuProfileTag = GpuProfileTag {
     color: debug_colors::BLACK,
 };
 
+trait TransformBatchKindMethods {
+    fn debug_name(&self) -> &'static str;
+    fn gpu_sampler_tag(&self) -> GpuProfileTag;
+}
+
+impl TransformBatchKindMethods for TransformBatchKind {
```

Comment:
can we just do `impl TransformBatchKind` here instead of adding a trait?

Reply:
Thanks for taking a look at this! I used a trait here, because TransformBatchKind is defined in tiling.rs.

Reply:
It's still the same crate. Notice how `BatchKind` is also not defined in this module but still implemented.

Reply:
Ah, you're right. I'll fix this.